src/workloads/scale/LLTMixed.yml

```diff
@@ -337,8 +369,57 @@ Actors:
         Database: *dbname
         Duration: *SnapshotScannerMediumDuration
         Blocking: none
-        Operations: *Operations
+        Operations: *InsertOps
 
+- Name: Low.Medium.Query
```

Comment:
How come this is only in one phase but the previous was in 2 phases?

Reply:
Baseline is only for the "short" duration for both low and high load. So all "shorts" are in two phases (without and with scanner). "Mediums" and "longs" are in one phase.